android/src/main/java/com/twiliorn/library/CustomTwilioVideoView.java

```diff
@@ -214,7 +213,7 @@ public class CustomTwilioVideoView extends View implements DefaultLifecycleObser
     private static PatchedVideoView thumbnailVideoView;
     private static LocalVideoTrack localVideoTrack;
 
-    private static CameraCapturer cameraCapturer;
+    private static FrameCaptureCameraCapturer cameraCapturer;
```

Comment:
See below for implementation of our custom CameraCapturer which can capture individual frames.